### ISSUE TITLE
Default .ssb/config?

### ISSUE THREAD

**Issue by ghost** [noffle/ssb-npm-101#1]
Hey @noffle I've been looking all over and I can't find where a skeleton config file might be found. My ~/.ssb/ doesn't have a config file and the sbot manual doesn't mention it. `sbot --help` didn't help either. 

I know this isn't the sbot repo but since it's the only place I've seen changes to the default config mentioned, could you point me in the right direction of that default config? Thank you!

**Comment by hackergrrl** [comment]
@wrycode what is your operating system? Do you have ssb installed and have run it already once before? `~/.ssb` is for unix systems -- macOS and Windows store application configuration files elsewhere.

**Comment by ghost** [comment]
Yes, I have the ~/.ssb directory and I've already run patchwork and sbot. There's no ~/.ssb/config file :(

**Comment by hackergrrl** [comment]
You can create it: it's a regular JSON file.
